crates/pop-parachains/src/utils/git.rs

```diff
@@ -27,10 +30,9 @@ impl Git {
 		}
 		Ok(())
 	}
-
 	pub(crate) fn ssh_clone(url: &Url, working_dir: &Path, branch: Option<&str>) -> Result<()> {
 		// Change the url to the ssh url with git@github.com: prefix, remove / from path and adding .git as suffix
-		let ssh_url = ["git@github.com:", &url.path()[1..], ".git"].concat();
+		let ssh_url = [Self::GIT_SSH, &url.path()[1..], ".git"].concat();
```

Comment:
Strictly speaking this is currently a github url rather than git, so the function should probably take it as a parameter so that the function can be used by any git provider. Probably doesnt matter due to the function being non-public.